pkgs/stdenv/darwin/default.nix

```diff
@@ -188,7 +188,15 @@ in rec {
   };
 
   stage1 = prevStage: let
-    persistent = _: super: { python = super.python.override { configd = null; }; };
+    persistent = self: super: with prevStage; {
+      cmake = super.cmake.override {
```

Comment:
Could you verify that this still works on cross? I can't remember the exact reason but we had to do ```super.buildPackages.cmake``` to avoid cross compilation issues.

Reply:
I don't fully understand what buildPackages implies in this context.

Reply:
This is fine @matthewbauer because the bootstrapping is explicitly native. We don't need to worry about persisting things with a target platform in the final stage, as if the host platform changes stdenv overrides are already chucked out. `cmake` has no target platform, and this is also an early stage platform, so we are good.